tests/integrations/gcp/test_gcp.py

```diff
@@ -93,8 +85,7 @@ def init_sdk(timeout_warning=False, **extra_init_args):
 @pytest.fixture
 def run_cloud_function():
     def inner(code, subprocess_kwargs=()):
-        events = []
-        envelopes = []
+        envelope_items = []
```

Comment:
Same comment here regarding naming -- we could keep calling this `envelopes` in the name of consistency with other tests